src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -158,7 +158,9 @@ public static async Task OutWriteAndWriteLineOverloads()
                 Console.SetOut(sw);
                 TextWriter writer = Console.Out;
                 Assert.NotNull(writer);
-                Assert.NotEqual(writer, sw); // the writer we provide gets wrapped
+                // Browser bypasses SyncTextWriter for faster startup
+                if (!OperatingSystem.IsBrowser())
```

Comment:
this should use `PlatformDetection.IsThreadingSupported` too instead of checking for browser